Storage/FileAccessor.swift

```diff
@@ -90,4 +90,31 @@ public class FileAccessor {
     private func createDir(absolutePath: String, error: NSErrorPointer = nil) -> Bool {
         return NSFileManager.defaultManager().createDirectoryAtPath(absolutePath, withIntermediateDirectories: true, attributes: nil, error: error)
     }
+
+    /**
+    * Writes the given data to a file at the given path. If the directory doesn't exist, 
+    * then create it.
+    */
+    public func write(relativePath: String, data: NSData?, error: NSErrorPointer = nil) -> Bool {
```

Comment:
The `data` argument should not be an optional. Makes no sense to write `nil` data. Better to catch this in the caller, and not let the caller even supply a `nil` `NSData` than to make it a special case here.